v3/src/lib/logger.ts

```diff
@@ -15,6 +15,7 @@ export interface LogMessage {
   application: string
   activity?: string
   event: string
+  event_value?: Record<string, unknown>
```

Comment:
It's not clear to me why we need another `Record<string, unknown>` property (`event_value`) when we already have one (`parameters`) that is essentially (or completely?) unused.

Reply:
In V2, it looks like `event_value` were variable values passed in to the log function, and `parameters` is `event_value` as an object. https://github.com/concord-consortium/codap/blob/cf79d90f7058fa9f913658de80d58bef04362a4f/apps/dg/controllers/authorization.js#L73.
And then the body has both: https://github.com/concord-consortium/codap/blob/cf79d90f7058fa9f913658de80d58bef04362a4f/apps/dg/controllers/authorization.js#L121

If we know nobody is using one over the other, we can remove event_value.